project/npda/templates/npda/visit_form.html

```diff
@@ -40,310 +40,10 @@
-                              <div class="basis-1/3 flex">
-                                <button type='button'
-                                        _="on click set #{{ field.id_for_label }}'s value to '{% today_date %}'"
-                                        class="btn rcpch-btn bg-rcpch_light_blue border border-rcpch_light_blue hover:bg-rcpch_strong_blue hover:border-rcpch_strong_blue {% if not can_alter_this_audit_year_submission or not can_complete_questionnaire %}opacity-40 pointer-events-none{% endif %}">
-                                  Today
-                                </button>
-                                <button type='button'
-                                        _="on click set #{{ field.id_for_label }}'s value to #id_visit_date.value"
-                                        class="btn rcpch-btn bg-rcpch_light_blue border border-rcpch_light_blue hover:bg-rcpch_strong_blue hover:border-rcpch_strong_blue {% if not can_alter_this_audit_year_submission or not can_complete_questionnaire %}opacity-40 pointer-events-none{% endif %}">
-                                  Fill with Visit Date
-                                </button>
-                              </div>
-                            {% else %}
-                              <input type="text" id="{{ field.id_for_label }}" name="{{ field.html_name }}" class="input rcpch-input-text rounded-none {% if not can_alter_this_audit_year_submission or not can_complete_questionnaire %}opacity-40 pointer-events-none{% endif %}" {% if field.field.required %}placeholder="Required"{% endif %} {% if field.value %} value={{ field.value }} {% elif field.value == 0 %} value="0.0" {% endif %}>
-                            {% endif %}
-                            {% for error in field.errors %}
-                              <p>
-                                <strong class="text-gray-700">{{ error|escape }}</strong>
-                              </p>
-                            {% endfor %}
-                          </div>
-                        </div>
-                      {% endif %}
-                    {% endfor %}
-                    {% if field_category in categories_with_errors %}
-                      <div>
-                        <p class="text-rcpch_red_dark_tint italic font-bold">Errors for category:</p>
-                        <p class="text-rcpch_red_dark_tint italic">
-                          {{ field_category|errors_for_category:visit_instance.errors|safe|linebreaksbr }}
-                        </p>
-                      </div>
-                    {% endif %}
-                  </div>
-                {% endif %}
-              {% endwith %}
-            {% endfor %}
-          </div>
+          {% include 'partials/visit_form_tab.html' with checked="true" label="Routine Measurements" categories="Measurements,HBA1c,Treatment,CGM,BP" %}
```

Comment:
It's not very easy to pass a list of parameters to a partial so I've jankily used comma separated values. I think it's worth it to avoid the copy paste - it should make it less likely we fix bugs in one place but forget in the others